test_regress/driver.py

```diff
@@ -738,7 +738,7 @@ def __init__(self, py_filename, scenario, running_id):
             ("+access+r" if Args.trace else "")
         ]
         self.nc_flags2 = []  # Overridden in some sim files
-        self.nc_run_flags = ["+licqueue", "-q", "+assert", "+sv", "-R"]
+        self.nc_run_flags = ["+licqueue", "-q", "+assert", "+sv", "-R", "-covoverwrite"]
```

Comment:
Not what you want if you intend to run a bunch of ncsim tests w/ coverage simultaneously.  But I am not planning on doing this.

Currently `xcelium.d/` and now `cov_work/` are being created under `test_regress/`.  I could possibly relocate these things to `obj_nc/t_foo/` dirs for each test, but that feels like overkill at this point.  Also, we'd still want `-covoverwrite`.  Let me know if something else is preferred.